test_scripts/API/VehicleData/HandsOffSteering/008_SubscribeVD_disallowed_after_PTU.lua

```diff
@@ -0,0 +1,71 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@GetmanetsIrina Please notice this script fails on https://github.ford.com/SmartDeviceLinkMirror/sdl_core/tree/feature/new_vehicle_data_hands_off_steering (9e169b289b) on all policy modes (P,E,H) and connection types (OFF, TCP, WS, WSS)

Reply:
The script is failed because of https://github.com/smartdevicelink/sdl_core/issues/3365